package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "monitor-utility",
-  "version": "2.0.3",
-  "description": "Utility for verbose outputs of Monitor data using a CLI",
```

Comment:
I don't think this is an intended change.

Reply:
Good catch. Only version was intended